frontend/src/styles/admin.css

```
@@ -197,7 +198,8 @@
 }
 
 .admin-page-header h1 {
-  font-family: Georgia, "Times New Roman", serif;
+  font-family: "Gowun Dodum", "Apple SD Gothic Neo", sans-serif;
+  font-weight: 700;
```

Stylelint (17.14.0)

[error] Duplicate property "font-weight" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)